src/main/java/net/datafaker/service/FakeValuesContext.java

```diff
@@ -69,7 +69,7 @@ private static String getFilename(Locale locale) {
      * This addresses that unfortunate condition.
      */
     private static String language(Locale l) {
-        if (l.getLanguage().equals("iw")) {
+        if ("iw".equals(l.getLanguage())) {
```

Comment:
Can this really return a null?

Reply:
this one probably not
however imho it's better to have constant first to avoid situations when someone uses this code as example however don't think about NPE

Reply:
Yeah, I don't agree, but that's fine. I rather be explicit about these things, instead of hiding the null behavior.